Add mailchimp_ecommerce_get_cart()

aaronbowwwls opened the pull request:
This fixes issue #144

Commit:
Add mailchimp_ecommerce_get_cart(), modeled after mailchimp_ecommerce_get_order().

taz77 commented on mailchimp_ecommerce.module:
```diff
@@ -699,6 +699,40 @@ function mailchimp_ecommerce_get_order($order_id) {
+ * @param string $cart_id
+ *   The cart ID.
+ *
+ * @return object
```
The return is mixed. An object or NULL can be returned.

aaronbowwwls replied:
Good call.

taz77 reviewed (approved):
Docblock could be adjusted for return.

Commit:
Include description parameter when calling mailchimp_ecommerce_update_product().

Commit:
Update docblocks for a couple functions.

mailchimp_ecommerce_get_cart() and mailchimp_ecommerce_get_order() can return either an object or NULL.

aaronbowwwls commented:
Issue #146 is fixed by commit fc8d53428274d1f76b88fd3cdafe6caac048ca71.
